pure-pso/values.yaml

```diff
@@ -80,27 +80,27 @@ flashblade:
   exportRules: "*(rw,no_root_squash)"
 images:
   plugin:
-    name: purestorage/k8s
-    tag: v6.2.2
+    name: pc2-dtr.dev.purestorage.com/purestorage/k8s
```

Comment:
why is this using pc2-dtr?

Reply:
not sure how it got missed, but it has been fixed now.